plotly/tests/test_optional/optional_utils.py

```diff
@@ -29,6 +29,25 @@ def _format_path(self, path):
         str_path = [repr(p) for p in path]
         return '[' + ']['.join(sp for sp in str_path) + ']'
 
+    def assert_fig_equal(self, d1, d2, msg=None, ignore=['uid']):
+        """
+        Helper function for asser_dict_equal
```

Comment:
typo `assert_dict_equal`

Reply:
It's not a typo. This is the wrapper function that removes keys in `ignore` then calls the good old `assert_dict_assert` we had before.

What's a better name for this?

How about I put this helper function inside `assert_dict_equal`, so:

```
def _dicts_with_removed_unwanted_keys(...):
   # does stuff

def assert_dict_equal(...):
   new_d1, new_d2  _dicts_with_removed_unwanted_keys(...)
   # the rest of the code
```

Reply:
@Kully I think Chris means the line, "Helper function for asser_dict_equal" (note "asser")

Reply:
thank you. corrected